localization/german/00_provinces_l_german.yml

```diff
@@ -0,0 +1,2279 @@
+﻿l_english:
```

Comment:
The contents of this file will not be referenced by CK3. It is a file that does not need to exist.